File: output/csharp/Modules/SnapshotAndRestore/Snapshot/GetSnapshot/GetSnapshotRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Modules {

	public class GetSnapshotRequest : RequestBase {
		
		[DataMember(Name="ignore_unavailable")]
		public bool IgnoreUnavailable { get; set; }


		[DataMember(Name="master_timeout")]
		public Time MasterTimeout { get; set; }


		[DataMember(Name="verbose")]
		public bool Verbose { get; set; }

	}
}

File: specification/specs/modules/snapshot_and_restore/snapshot/get_snapshot/GetSnapshotRequest.ts
@rest_spec_name("snapshot.get")
class GetSnapshotRequest extends RequestBase {
  path_parts?: {
    repository: Name;
    snapshot: Names;
  }
  query_parameters?: {
    ignore_unavailable?: boolean;
    master_timeout?: Time;
    verbose?: boolean;
  }
  body?: {
  }
}
